Fork of ExUTR by Zixia Huang (https://github.com/huangzixia/ExUTR/) correcting minor bugs and enforcing alternative design.
Further forked from Paul Andrey https://github.com/pandrey-fr/ExUTR for edits to made to allow this to work with a modern swissprot database and on the WVU supercomputer
All credits go to the initial author.


##ExUTR
###ChangeLog
Last update 16/08/2017
Created 15/02/2017
Version 0.1.0

###Introduction

ExUTR is a practical and powerful workflow that enables rapid genome-wide 3'-UTR prediction from massive RNA-Seq experiments. The implementation of ExUTR is based mainly on the intrinsic signals of transcripts, which is especially intriguing for countless non-model organisms where reference genomes and annotations are missing. Being independent of a well-annotated reference genome will dramatically expand its application to much broader research area for which RNA-Seq data is available.

ExUTR contains three Steps: 1) Transcriptome assembly (Optional); 2) ORF prediction; 3) 3'-UTR retrieval.

ExUTR has been tested on Ubuntu 12.04 desktop but it should work on most Linux operating systems.

#### - [Installation] (https://github.com/huangzixia/ExUTR/blob/master/doc/Installation.md)
#### - [Quick Start] (https://github.com/huangzixia/ExUTR/blob/master/doc/Quick_start.md)

###ExUTR Workflow

**Workflow.** 

#### - [Installation] (https://github.com/huangzixia/ExUTR/blob/master/doc/workflow.png)

###Comments and bug report

Please send your comment and bug reports to zixia.huang@ucdconnect.ie.

*Copyright (c) 2017, UCD School of Biology and Environmental Science, University College Dublin, Belfield, Dublin 4, Ireland.*
